spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -852,6 +916,16 @@ deployer.<app>.kubernetes.entryPointStyle=<Entry Point Style>
 
 Replacing `<app>` with the name of your app and the desired Entry Point Style.
 
+The image pull secret can also be configured at the server level in the container env section of a deployment YAML, for example:
```

Comment:
Maybe a copy + paste error. Perhaps this should specifically discuss the `entryPointStye`.